src/standalone/Soil/Biogeochemistry/Biogeochemistry.jl

```diff
@@ -581,7 +553,7 @@ function ClimaLand.boundary_flux(
 end
 
 function ClimaLand.get_drivers(model::SoilCO2Model)
-    return (model.driver.atmos, nothing)
+    return (model.driver.atmos, model.driver.soc)
```

Comment:
Consistency question: 
we have p.drivers (with an s) and model.driver (with no s)
is this on purpose?

Reply:
I noticed that in the previous PR, and I think it should be "drivers" everywhere, now! thanks for catching!